src/Jackalope/Transport/DoctrineDBAL/XmlParser/XmlToPropsParser.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace Jackalope\Transport\DoctrineDBAL\XmlParser;
+
+use PHPCR\PropertyType;
+use PHPCR\Util\ValueConverter;
+
+/**
+ * @internal
+ */
+class XmlToPropsParser
+{
+    /**
+     * @var string
+     */
+    private $xml;
+
+    /**
+     * @var string[]|null
+     */
+    private $propertyNames;
+
+    /**
+     * @var ValueConverter
+     */
+    protected $valueConverter;
```

Comment:
what is the use case of having this property be `protected` instead of `private`? i suggest to make this private as well and mark the whole class as `final`.

Reply:
copy paste error 😄 will change it.